src/test/java/com/github/jasminb/jsonapi/ResourceConverterTest.java

```diff
@@ -127,6 +127,13 @@ public void testWriteCollection() throws IOException, IllegalAccessException {
 
 		JSONAPIDocument<List<User>> usersDocument = converter.readDocumentCollection(usersRequest, User.class);
 		List<User> users = usersDocument.get();
+
+		assertNotNull(users);
+		assertEquals(2, users.size());
+
+		// Make sure that relationship object i.e. statuses is null
+		assertNull(users.get(0).getStatuses());
+
 		byte[] convertedData = converter.writeObjectCollection(users);
```

Comment:
Looks like JSON produced here holds an empty `statuses` relationship object and breaks the spec which states that it must have one of `links/da/data/meta`.

Reply:
Thanks. I will fix it asap.

Reply:
Request change has been incorporated.

Reply:
Hello, first of all apologies for a such long time that it has taken for me to take a look again.

Lib still produces empty relationship object for resources that are serialised and are lacking relationship details in the annotation. For example:

```java
	@Relationship(value = "statuses")
	private List<Status> statuses;
```

Produces:

```json
{
  "data": [
    {
      "type": "users",
      "id": "1",
      "attributes": {
        "name": "liz"
      },
      "relationships": {
        "statuses": {}
      },
      "links": {
        "self": "https://api.example.com/users/1"
      }
    },
    {
      "type": "users",
      "id": "2",
      "attributes": {
        "name": "john"
      },
      "relationships": {
        "statuses": {}
      },
      "links": {
        "self": "https://api.example.com/users/2"
      }
    }
  ]
}
```

It should omit `relationships` altogether as there are no non-empty elements in it:

```json
{
  "data": [
    {
      "type": "users",
      "id": "1",
      "attributes": {
        "name": "liz"
      },
      "links": {
        "self": "https://api.example.com/users/1"
      }
    },
    {
      "type": "users",
      "id": "2",
      "attributes": {
        "name": "john"
      },
      "links": {
        "self": "https://api.example.com/users/2"
      }
    }
  ]
}
```

Adding relationship annotation fixes the issue but thats besides the point as its not required.
```java
	@Relationship(value = "statuses", path = "/relationship/statuses")
	private List<Status> statuses;
```